src/mem/ruby/protocol/GPU_VIPER-TCC.sm

```diff
@@ -289,7 +290,9 @@ machine(MachineType:TCC, "TCC Cache")
             is_slc_set := tbe.isSLCSet;
         }
 
-        if (is_slc_set) {
+        if (in_msg.Type == CoherenceResponseType:NBSysWBAck) {
```

Comment:
Please add a comment here explaining what this is for

Reply:
Added.